modules/extensions/src/views/lite/components/debugger/views/Dialog.tsx

```diff
@@ -16,8 +15,7 @@ const Decision: SFC<{ decision: sdk.IO.Suggestion }> = props => (
   <div className={style.subSection}>
     <H5 color={Colors.DARK_GRAY5}>Decision</H5>
     <div style={{ display: 'flex' }}>
-      <Intent name={props.decision.sourceDetails} />
-      &nbsp;
+      <strong>{props.decision.sourceDetails}</strong>&nbsp;
```

Comment:
sourceDetails is good if the answer is a qna, you will be able to navigate to it. IT should just display raw text if it's "Execute default flow"